src/main/java/org/asamk/Signal.java

```diff
@@ -21,6 +21,14 @@ long sendMessage(
             String message, List<String> attachments, List<String> recipients
     ) throws Error.AttachmentInvalid, Error.Failure, Error.InvalidNumber, Error.UntrustedIdentity;
 
+    void sendTyping(
+            String recipient, Boolean stop
+    ) throws Error.Failure, Error.GroupNotFound, Error.UntrustedIdentity;
+
+    void sendReceipt(
+            String recipient, long targetSentTimestamp
```

Comment:
Should accept a `List<Long>` of timeframe timestamps to mark multiple messages as read at the same time